fetch.bs

```diff
@@ -2619,6 +2620,9 @@ boolean <var>http3Only</var>, run these steps:
 <a for=connection>timing info</a> and observe these requirements:
 
 <ul>
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">domain lookup end time</a> should be
+ the time after resolving the IP address of the server or proxy. [[SVCB]]
```

Comment:
It should be deleted from there, thought I'd done that :(
revising

Reply:
I see, we could make this imperative I think.

1. Have a step before "For each proxy of proxies:" that sets it.
2. Give the 'If proxy is "DIRECT"' conditional substeps one of which also sets it. (The other is the current substep.)
3. Now that it has substeps, the "continue" subsequent could be moved under it as well I think. _hosts_ cannot be failure for any other reason.

I think that's clearer than this. Since we're not really talking about resolving the IP address, we're instead interested in when the domain or proxy domain has been resolved.

Reply:
But the previous definition was wrong... we are interested in when the actual IP address was resolved and we don't want to expose anything proxy related.

Reply:
> My suggestion wouldn't do that, right? And what does it mean to resolve an IP address?

I mean that the step that resolves a (proxy/regular) host name to an IP address is any way not part of the proxy resolution steps. We can add those steps to get some default `domain lookup end time` but it would be redundant as we would still need the requirement prose

Reply:
Ah, looking again, you're right, I missed something.

Reply:
Fixed now, I think.

Reply:
you get an end time for each potential connection, which will be applied to the connection's connection info - and only the first successful one would be attached to the response.

Reply:
I'm lost. Why does the start time concern not apply to the end time? Or do neither apply and the current text is okay as-is?

Reply:
That delay would be hidden inside `domainLookupStart` and `domainLookupEnd`. The proxy time would be indistinguishable from DNS time.

Reply:
But this is different from the intent of the earlier proposed text

>  timingInfo's domain lookup end time should be the time after resolving the IP address of the server or proxy.

right? I thought that the end time should either be after finding a proxy (and resolving its host) or after resolving a host (in case there's no proxy).

One more question: should these times match when the input is an IP address?

Reply:
Right, the problem is that in the case of proxies, it's not specified when the proxy host is resolved to a proxy IP address. Hence the earlier requirement-style text.

Reply:
I see, I kinda thought that would be part of 

> Let proxies be the result of finding proxies for url in an [implementation-defined](https://infra.spec.whatwg.org/#implementation-defined) manner.

but I agree that is not defined at the moment.

What about the case where there's no domain lookup because we start with an IP address?

Reply:
This actually needs to be discussed, there might be some existing privacy issue here.